job_runner/reporting/misc.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC, abstractmethod
+
+
+class Report(ABC):
+    @abstractmethod
+    def report(self, msg):
+        raise NotImplementedError()
```

Comment:
Not really needed on `@abstractmethod`, could use `pass` or add a description in a doc string.

Reply:
removed